apps/trustlab/src/payload/collections/Pages.js

```diff
@@ -0,0 +1,42 @@
+import TestBlock from "@/trustlab/payload/blocks/TestBlock";
+import fullTitle from "@/trustlab/payload/fields/fullTitle";
+import slug from "@/trustlab/payload/fields/slug";
+
+const Pages = {
+  slug: "pages",
+  access: {
+    read: () => true,
+    create: () => true,
+    update: () => true,
```

Comment:
We'll need different author, editor access so lets add an issue for it (if one doesn't exist already)

Reply:
#1136